### ISSUE TITLE
missing change replication factor and partition number in topic configuration

### ISSUE THREAD

**Issue by raphaelauv** [redpanda-data/console#964]
there is no way to change the replication factor and partition number of topics 

this feature is present in provectus in the topic configuration

![image](https://github.com/redpanda-data/console/assets/10202690/9cf3da1b-74e5-4857-a95d-ec6b98f20a60)

I would enjoy that feature in redpanda console , thanks all

**Comment by weeco** [comment]
Makes sense, I'd like to see this as well in Console and I'm sure we can provide a bit more guidance for a better UX :-)
